media-libs/libopenmpt/libopenmpt-0.7.10.recipe

```diff
@@ -3,16 +3,16 @@ DESCRIPTION="libopenmpt is a cross-platform C++ and C module playback \
 library. It is based on the player code of the Open ModPlug Tracker project. \
 It offers high-quality module playback on a variety of platforms."
 HOMEPAGE="https://openmpt.org/"
-COPYRIGHT="2004-2023 OpenMPT contributors
+COPYRIGHT="2004-2024 OpenMPT contributors
 	1997-2003 Olivier Lapicque"
 LICENSE="BSD (3-clause)"
 REVISION="1"
 SOURCE_URI="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-$portVersion+release.autotools.tar.gz"
-CHECKSUM_SHA256="509143e5a4f3b13dd98eae289bfec1b564eddd71bbc7ca7864b82835e31330e1"
+CHECKSUM_SHA256="093713c1c1024f4f10c4779a66ceb2af51fb7c908a9e99feb892d04019220ba1"
 SOURCE_DIR="libopenmpt-$portVersion+release.autotools"
 
 ARCHITECTURES="all !x86_gcc2"
-SECONDARY_ARCHITECTURES="!x86_gcc2 x86"
+SECONDARY_ARCHITECTURES="x86"
 
 libopenmptVersion=0.4.4
```

Comment:
I would assume that the version has probably changed. Have you checked this?

Reply:
Yes, no change. LIB versions usually don't change for minor releases.